packages/docusaurus-init/templates/classic/docs/gettingStarted.md

```diff
@@ -0,0 +1,71 @@
+---
+id: getting-started 
+title: Getting Started 
+slug: /
+---
+
+import TOCInline from '@theme/TOCInline';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page will help you to get started with the development process of your Docusaurus project.
+
+
+## Setting up the development environment
+
+If you haven't setup your development environment. Let's go through a small step-by-step tutorial to get started with your Docusaurus project.
+
+:::note
+Feel free to edit this page as you follow the tutorial!
+:::
+
+### Step 1:  Generate a Docusaurus site
+
+Let's run a command to generate the classic template. You can run this command anywhere either in a new empty repository or within an existing repository.
+
+```shell
+npx @docusaurus/init@latest init my-documentation classic
+```
+
+### Step 2: Install Dependencies
```

Comment:
not 100% sure but I think this  part is not necessary, as the "init" command already install the dependencies

Reply:
Also, I think it may be preferable if all the MDX items of the tutorial like tabs were contained in a single doc like `markdown-features.mdx`.

I think the very first doc the user edit should be simple and not contain JSX syntax